src/run/serve.rs

```diff
@@ -109,10 +111,19 @@ pub(crate) fn serve(web_bundle: WebBundle, port: u16) -> anyhow::Result<()> {
                             // Try to inject the auto reload script in the document body
                             // TODO: Do this also for the other cases when the `index.html` is in a
                             // folder
-                            let contents = contents.replace(
+                            let mut contents = contents.replace(
                                 "</body>",
                                 r#"<script src="_bevy_dev/auto_reload.js"></script></body>"#,
                             );
+                            if !contents.contains("</title>") {
+                                contents = contents.replace(
```

Comment:
This currently only happens for the dev server version, not for the bundles or if the user provided their own `index.html` without `<title>`.

I'll check if I can insert this elsewhere to make it more broadly available.